### DISCUSSION TITLE
Should archaic English be modernized?

### DISCUSSION THREAD

**Discussion by fiapps** [DivinumOfficium/divinum-officium#2708] [General]
I occasionally receive reports of supposed mistranslations that are actually just archaic expressions. Here are a couple of examples:

1. "intácta nésciens virum" in Responsory5 of Sancti/01-01 is translated "The stainless and unwedded one." So a user complains that Mary was not unwedded, but espoused to Joseph (matrimonium ratum et non consummatum, according to Jewish custom). I can explain that Mary is called "unwedded" in translations of the Fathers and liturgical texts, e.g., the "unwedded bride" of the Akathistos, because she never consummated her marriage. However, the expression is confusing to the uninitiated, and I couldn't even find this meaning of unwedded in the [OED](https://archive.org/details/in.ernet.dli.2015.271837/page/n1373/mode/1up) (1st ed.).
2. I don't recall where it was reported, but a user complained about a text like "For thou hast prevented him with the blessings of goodness" (Sancti/09-28) being a clumsy mistranslation by a false cognate. This sense of "prevent" is at least in the dictionary, which made the translation easier to explain, but this kind of translation is likely to be misunderstood.
3. St. Austin, an archaic form of St. Augustine, has been reported to me as an error a couple of times.

**Comment by APMarcello3** [comment]
I have no objection to updating these archaisms. If you could provide a list, I will be happy to change them. I believe the texts should be faithful to the Latin but at the same time readable and intelligible.

**Comment by fiapps** [comment]
I started by looking for archaic uses of "prevent," but a lot of them are in the Psalms, and I don't know if we want to change the Douay translation.

### Psalms
* 17:6 The sorrows of hell encompassed me: * and the snares of death prevented me.
* 17:19 They prevented me in the day of my affliction: * and the Lord became my protector.
* 20:4 For thou hast prevented him with blessings of sweetness: * thou hast set on his head a crown of precious stones.
* 58:10 I will keep my strength to thee: for thou art my protector: * my God, his mercy shall prevent me.
* 76:5 My eyes prevented the watches: * I was troubled, and I spoke not.
* 78:8 Remember not our former iniquities: let thy mercies speedily prevent us, * for we are become exceeding poor.
* 87:14 But I, O Lord, have cried to thee: * and in the morning my prayer shall prevent thee.
* 118:147 I prevented the dawning of the day, and cried: * because in thy words I very much hoped.
* 118:148 My eyes to thee have prevented the morning: * that I might meditate on thy words.

Then we have Psalm 20:4 used in C2 and in some proper responsories.

`web/www/horas/English/Commune/C2.txt`
V. For Thou hast prevented him with the blessings of sweetness: Thou hast set a crown of precious stones upon his head.

`web/www/horas/English/Sancti/01-16.txt`
R. O Lord, thou hast prevented him with blessings of sweetness:

`web/www/horas/English/Sancti/02-11.txt`
R. Thou hast prevented her, O Lord, with blessings of sweetness, thou hast set on her head 

`web/www/horas/English/Sancti/07-08t.txt`
V. For thou hast prevented her with the. blessings of sweetness.

`web/www/horas/English/Sancti/09-28.txt`
V. For thou hast prevented him with the blessings of goodness, and hast set a crown of pure gold upon his head.

### Other quotations from the Douay-Rheims

`web/www/horas/English/Commune/C5.txt`
7 But the just man, if he be prevented with death, shall be in rest.

`web/www/horas/English/Sancti/01-08.txt`
10 Loving one another with the charity of brotherhood, with honour preventing~

### Other usage of prevent

`web/www/horas/English/Sancti/01-21.txt`
Depart from me, prey of death, for I am prevented by another Lover.;;1

`web/www/horas/English/Sancti/10-27.txt`
Almighty God, grant, we beseech thee, that as we are preventing thine Apostles~
Simon and Jude, their glorious birth - day, so the same may prevent thy Majesty~

`web/www/horas/English/Sancti/12-14.txt`
as to thee, to be prevented by purifying grace; to none, as to thee, to shine~

`web/www/horas/English/Tempora/Pent16-0.txt`
Lord, we pray thee, that thy grace may always prevent and follow us, and make us continually to be given to all good works.

**Comment by fiapps** [comment]
Unwedded is less common.

* One case is that reported above, in `Responsory5` and `Responsory8` of `www/horas/English/Sancti/01-01.txt`, "intácta nésciens virum" is translated "The stainless and unwedded one" for the sake of rhyme.
* The other case is in a passage from a pseudo-Augustine sermon 194 (PL 39:2106) that is used for `Lectio M021` in `web/www/horas/English/CommuneM/C10.txt` and also for `Lectio4` in `web/www/horas/English/Sancti/09-09t.txt`. Here "unwedded" is the obvious translation of "innupta," but "unwedded mother" can give the wrong impression today. Perhaps: "She is not frightened to give birth without having consummated marriage, …"

**Reply by ajmalton** [reply]
I imagine there are some users who actually pray the vernacular texts, even if (most) others use to help understand the canonical text. Some may then prefer the hieratic language, at least in English. And others may prefer not to change and “modernize” anyway.

I think modernizing our translations is a gargantuan, elephantine, leviathan task -- and so all we would actually manage to do is to change a few words here and there, ad hoc, with dispute before and after. The question of “unmarried” multiplied ten thousand times over. I foresee far more heat than light.  

(A smaller analogous task is making the “rubrics” in the translations actually be translations of the canonical rubrics, which they usually aren't. I briefly began this, in just one area of the Missal, and only for the languages I know. And even then I quickly saw that it's a bad idea, essentially impossible to do well unless very slowly and carefully.)

In my view (which need not prevail) it were better to leave the translations alone. At the very most, let the versions we now have become bases for new versions, while keeping the old ones; and release the new ones only when they're different enough to serve their purpose.

**Reply by fiapps** [reply]
There certainly are people who pray the vernacular texts, and some do prefer language resembling the Book of Common Prayer. When I first released my app a decade ago, there was a comment from someone who was very pleased that the translations didn't use American spelling. More often, I've received complaints about thees, thous, and archaic usage that looks like a mistranslation. On the other hand, people are more likely to complain about a perceived problem than express appreciation for what they feel has been done well.

In any case, Fr. Marcello has been making changes to the translations for years now, changing a word here, replacing the translation of an oration there. I don't think it's a question any more of whether to change it, but how much and on the basis of what principles.

**Reply by fiapps** [reply]
I got a message a few days ago from someone who strongly prefers the old translation, and doesn't want to use my app until it's available again. It would be possible and fairly straightforward to preserve the old translation as English-Bute (for Marquess of Bute), but I don't have the time to create such a thing. If anyone else does, perhaps https://github.com/fiapps/officiumprog will be useful to him. It contains some of the last versions Laszlo Kiss released before his death, so it should be almost all the original translation. Of course, it also has all the typos that have been fixed over the last 11 years.

**Comment by fiapps** [comment]
PR #2711 changes Austin to Augustine. I didn't change any of the surrounding text, so "holy Austin" becomes "holy Augustine", not "St. Augustine."
